Cogs.Publishers/ModelInitializer.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Cogs.Publishers
+{
+    public class ModelInitializer
+    {
+        public string Dir { get; set; }
+
+        public void Create()
+        {
+            if (Directory.Exists(Dir))
+            {
+                throw new InvalidDataException("The directory already exist");
+            }
+            //Create the directory that user want
+            DirectoryInfo di = Directory.CreateDirectory(Dir);
+
+            //Create the 4 major folders and 1 readme file
+            DirectoryInfo it = Directory.CreateDirectory(Path.Combine(di.FullName, "ItemTypes"));
+            DirectoryInfo rt = Directory.CreateDirectory(Path.Combine(di.FullName, "ReusableTypes"));
+            DirectoryInfo setting = Directory.CreateDirectory(Path.Combine(di.FullName, "Settings"));
+            DirectoryInfo topics = Directory.CreateDirectory(Path.Combine(di.FullName, "Topics"));
+            File.WriteAllText(Path.Combine(di.FullName, "readme.md"), "Model description");
+
+            //Create itemtype
+            DirectoryInfo i1 = Directory.CreateDirectory(Path.Combine(it.FullName, "item1"));
+            DirectoryInfo i2 = Directory.CreateDirectory(Path.Combine(it.FullName, "item2"));
+
+            //write to the directory
+            //create item1
+            StringBuilder item1 = new StringBuilder();
+            StringBuilder item1readme = new StringBuilder();
+            item1.AppendLine("Name,DataType,MinCardinality,MaxCardinality,Description,MinLength,MaxLength,Enumeration,Pattern,MinInclusive,MinExclusive,MaxInclusive,MaxExclusive,DeprecatedNamespace,DeprecatedElementOrAttribute,DeprecatedChoiceGroup");

```

Comment:
Create a Property object, and set these sample properties programmatically. 
Then use the CsvWriter WriteRecords method to write them as a csv file.

Build properties like this: https://github.com/Colectica/DdiToCogs/blob/master/DdiToCogs/ConvertToCogs.cs#L878
Write the csv file like this: https://github.com/Colectica/DdiToCogs/blob/master/DdiToCogs/ConvertToCogs.cs#L716

Do this wherever we write out example csv files